### ISSUE TITLE
[FEATURE] add light mode

### ISSUE THREAD

**Issue by Dun-sin** [Dun-sin/template-portfolio#3]
### Description

The project just as a dark mode

### Additional information

Tasks:
- [ ] Add a toggle to switch between light and dark mode
- [ ] Configure the colours according to the mode chosen

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/template-portfolio/blob/main/CONTRIBUTING.md)

**Comment by Dun-sin** [comment]
> Hi I'd like to pick this up

Definitely, thanks for wanting to contribute, Make sure you follow the rules [👉🏽here👈🏽](https://github.com/Dun-sin/template-portfolio/blob/main/CONTRIBUTING.md#rules), else your PR will not be accepted and will be closed. 
Good Luck

**Comment by Daniel-Ezekiel** [comment]
@Dun-sin Hi, I was wondering, is it possible for me to take this issue and work on it? It looks like it has been pending for a while, I hope this request is not coming late.

**Comment by Dun-sin** [comment]
> @Dun-sin Hi, I was wondering, is it possible for me to take this issue and work on it? It looks like it has been pending for a while, I hope this request is not coming late.

It's not, would appreciate you doing it🙏🏽

**Comment by Daniel-Ezekiel** [comment]
Thanks. I will get on it. Hopefully I am able to complete it quickly.

By the way, just to clarify, may I choose the light mode colors myself? Although, I intend to keep it within scope by using the colors that are in the dark mode as well and I also know that the outcome should not deviate much from the original.

**Comment by Dun-sin** [comment]
> Thanks. I will get on it. Hopefully I am able to complete it quickly.
> 
> By the way, just to clarify, may I choose the light mode colors myself? Although, I intend to keep it within scope by using the colors that are in the dark mode as well and I also know that the outcome should not deviate much from the original.

Sure, just make it clean and presentable

**Comment by Daniel-Ezekiel** [comment]
> 

Yeah, I saw the video but I'm asking because what I noticed in the video is that what is in the PR template file, but I did not notice anything similar to what I was seeing in the Issues folder.
